modules/slider/README.md

```diff
@@ -0,0 +1 @@
+## @diana-ui/slider
```

Comment:
As @diogo-mendes likes to remind 😆 , write here some description of component. Something small